test/audits/cwv.test.js

```diff
@@ -15,11 +15,15 @@
 import { expect, use } from 'chai';
 import sinon from 'sinon';
 import sinonChai from 'sinon-chai';
+import chaiAsPromised from 'chai-as-promised';
 import nock from 'nock';
-import { CWVRunner } from '../../src/cwv/handler.js';
-import { rumData } from '../fixtures/rum-data.js';
+import { CWVRunner, convertToOppty } from '../../src/cwv/handler.js';
+import expectedOppty from '../fixtures/cwv/oppty.json' assert { type: 'json' };
```

Comment:
Hi, I am sure you have noticed, but just in case:
asserts are deprecated and no longer works on nodejs v22+.
https://stackoverflow.com/questions/78876691/syntaxerror-unexpected-identifier-assert-on-json-import-in-node-v22